org/ystia/topologies/a4c_yorc_docker_basic/README.md

```diff
@@ -0,0 +1,99 @@
+# Simple Yorc stack application template
+
+Defines an application template that alows to deploy a Yorc Stack composed by Yorc&Consul + Alien4Cloud.
+The **A4C** used to deploy the application has version 2.2.0-SM10
+
+Application is composed by the following components that need to already be uploaded in the A4C catalog:
+- org.ystia.yorc.docker
+- org.ystia.alien.docker
+
+## Prerequisits
+
+An available Kubernetes cluster.
+A Namespace context set (otherwise the Yorc stack is deployed on the **default** namespace).
+A ConfigMap created using --from-file option equal to the path of a directory that contains the configuration files for the Yorc to be deployed.
+
+## Usage
+
+### Update A4C Catalog
+
+Check used versions in this template and in components types, and if necessary modify them correspondingly to your needs. 
+Create a zip file with the types.yaml and upload it to the A4C's Catalog.
+
+### Create an application using this template.
+
+### Deploy it to a K8S location configured within a Yorc orchestrator (YO).
+See below for K8S location configuration
+
+### Check workloads and services on K8S
+
+```
+$ kubectl get services
+
+alien-aliendeployment-service-786916228   LoadBalancer   10.3.241.199   35.240.85.70   8088:32564/TCP
+yorc-yorcdeployment-service--1179116320   NodePort       10.3.255.250   <none>         8800:32018/TCP,8500:30449/TCP
+
+```
+
+### Configure deployed components
+
+Currently a manual configuration is necessary at this step.
+
+Some informations are necessary to be get from K8S. 
+
+In order to connect to the deployed Alien4Cloud's GUI:
+* alien-aliendeployment-service name; here alien-aliendeployment-service-786916228
+* alien-aliendeployment-service Load balancer IP ; here 35.240.85.70
+* alien-aliendeployment-service alien-console port; here 8088
+The Alien4Cloud's GUI can be reached with http://<lb-ip>:<alien-console-ip>; here  http://35.240.85.70:8088
+
+In order to connect the Alien4Cloud to the deployed Yorc orchestrator:
+* yorc-yorcdeployment-service name; here yorc-yorcdeployment-service--1179116320
+* yorc-yorcdeployment-service ClusterIP (NodePort Cluster IP); here 10.3.255.250
+* yorc-yorcdeployment-service yorc-server port; here 8800
+The Yorc orchestrator's URL will be http://<cluster-ip>:<yorc-server-port> ; here http://10.3.255.250:8800
+
+Now follow the next steps:
+
+1. Connect to the deployed Alien Console : http://35.240.85.70:8088
+2. Create a Yorc orchestrator (Yorch) with Yorc URL http://10.3.255.250:8800
```

Comment:
Why don't use the service name here instead of the cluster IP ?

Reply:
Yes, you are right. I fix this.